File: server-api/src/main/java/com/ftn/app/serverapi/model/Laptop.java
package com.ftn.app.serverapi.model;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "laptop")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class Laptop {

    @Id
    private Long id;
    private String brand;
    private String processorBrand;
    private String processorModel;
    private String cores;
    private String ramGeneration;
    private Integer ramAmount;
    private String storageType;
    private Integer storageAmount;
    private String screenSize;
    private Integer price;
    private String condition;
    private String url;

}

File: scripts/insert/model/laptop.py
class Laptop:
	def __init__(self, laptop_brand, processor_brand, processor_number, number_of_cores, ram_generation, ram_amount, storage_type, storage_amount, screen_size, price, condition, url, laptop_id):
		self.laptop_brand = laptop_brand
		self.processor_brand = processor_brand
		self.processor_number = processor_number
		self.number_of_cores = number_of_cores
		self.ram_generation = ram_generation
		self.ram_amount = ram_amount
		self.storage_type = storage_type
		self.storage_amount = storage_amount
		self.screen_size = screen_size
		self.price = price
		self.condition = condition
		self.url = url
		self.laptop_id = laptop_id

	def display(self):
		return "laptop_id", self.laptop_id
